bats/tests/containers/switch-engines.bats

```diff
@@ -0,0 +1,56 @@
+# Test case 20
+
+setup() {
+    load '../helpers/load'
+    RD_CONTAINER_ENGINE=moby
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start container engine' {
+    start_container_engine
+    wait_for_container_engine
+}
+
+@test "pull nginx" {
+    docker run -d -p 8085:80 --restart=no nginx
+    run docker ps --format '{{json .Image}}'
+    assert_output --partial nginx
+}
+
+@test "switch to containerd" {
+    RD_CONTAINER_ENGINE=containerd
+    run rdctl set --container-engine.name=containerd
+    assert_success
+    wait_for_container_engine
+    nerdctl run -d -p 8086:80 --restart=no nginx
+    run nerdctl ps --format '{{json .Image}}'
+    assert_output --partial nginx
+}
+
+@test 'switch back to moby' {
+    RD_CONTAINER_ENGINE=moby
+    run rdctl set --container-engine.name moby
+    assert_success
+    wait_for_container_engine
```

Comment:
```suggestion
    set_container_engine moby
```

Reply:
Refactored the thing down to the factory-reset test and four other specific tests